typescript/t.d.ts

```diff
@@ -205,6 +205,19 @@ export type KeyWithContext<Key, TOpt extends TOptions> = TOpt['context'] extends
   ? `${Key & string}${_ContextSeparator}${TOpt['context']}`
   : Key;
 
+export type ContextOfKey<
```

Comment:
This is used only on this file so I think we can remove the `export`
```suggestion
type ContextOfKey<
```